packages/instant-apps-components/src/components/instant-apps-create/instant-apps-create.test.ts

```diff
@@ -0,0 +1,86 @@
+import { expect, test, describe } from 'vitest';
+import '../../../dist/components/instant-apps-create.js';
+import WebScene from '@arcgis/core/WebScene';
+import config from '@arcgis/core/config';
+
+config.portalUrl = 'https://nw-brews.mapsdevext.arcgis.com';
+
+const scene = new WebScene({
+  portalItem: {
+    id: '0614ea1f9dd043e9ba157b9c20d3c538', // ID of the WebScene on the on-premise portal
+  },
+});
+describe('inline', async () => {
+  const create = document.createElement('instant-apps-create');
+  create.mode = 'inline';
+  create.setAttribute('show-scale-bar', 'true');
+  create.setAttribute('id', 'create-inline');
+  test('test if create option map works in inline', async () => {
+    create.portal = scene.portalItem.portal;
+    create.content = scene;
+
+    document.body.appendChild(create);
+
+    await new Promise(resolve => requestIdleCallback(resolve));
+    await new Promise(resolve => setTimeout(resolve, 5000));
+
+    const shadowRoot = document.getElementById('create-inline')?.shadowRoot;
```

Comment:
the node is stored in a const on line 14, so it doesn't seem like querying for the node is necessary